personalsite/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { MenuItems } from "./MenuItems"
+import { Button } from "../Button"
+import './Navbar.css'
+
+class Navbar extends Component {
+    state = { clicked: false }
+
+    handleClick = () => {
+        this.setState({ clicked: !this.state.clicked })
+
+    }
+
+    
+
+    render() {
+
+        const Navbar = (props) => {
+
+            const displayComponent1 = () =>{
+                props.changeCurrPage(1)
+            }
+    
+            const displayComponent2 = () =>{
+                props.changeCurrPage(2)
+            }
+        }
+
+        return(
+            <nav className="NavbarItems">
+                <h1 className="navbar-logo">Veronica<i className="fab fa-react"></i></h1>
+                <div className="menu-icon" onClick={this.handleClick}>
+                    <i className={this.state.clicked ? 'fas fa-times' : 'fas fa-bars'}></i>
+
+
+                </div>
+                <ul className={this.state.clicked ? 'nav-menu active' : 'nav-menu'}>
+                    {MenuItems.map((item, index) => {
```

Comment:
so the reason its not changing pages on clicking the navbar is because you are not implementing it on click here. So for each menu item you could do 

```javascript
return(
<li key = {index}>
   <div onClick = {() => {this.changePage(item.title)}> {item.title} </div>
</li>
);
```

and then you could have a function above:
```javascript
changePage = (page) => {
   props.changeCurrPage(page)
}

```

this would change the state in App.js to whatever the title of the navbar button is. and then you can conditionally render based on page name